test/fixtures/hello-cloud/configmap-runner.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hello-cloud-configmap-runner
+  labels:
+    name: hello-cloud-configmap-runner
+    app: hello-cloud
+data:
```

Comment:
So pod template == ConfigMap?
https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/#pod-templates doesn't say anything about config maps. Maybe documenting it in the README would help to make it clear.